.docksal/docksal.yml

```diff
@@ -1,6 +1,11 @@
 version: "2.1"
 
 services:
+  # Xdebug setup for cli
+  cli:
+    environment:
+      - PHP_IDE_CONFIG=serverName=${VIRTUAL_HOST}
```

Comment:
This is already set in the default stack:
https://github.com/docksal/docksal/blob/4ded52a756c2edff6af222f427a067c712b1ebce/stacks/services.yml#L148

Reply:
Curious--that's what I'd thought too but it wasn't in my local--I'll dig into that, I might have borked it somehow.